app/api/volunteers/special_cases_api.rb

```diff
@@ -0,0 +1,94 @@
+module Volunteers
+  class SpecialCasesApi < Grape::API
+    use Grape::Knock::Authenticable
+
+    before { authorize_user_role! }
+
+    resource :special_cases do
+      desc "My special cases" do
+        tags %w[special_cases]
+        http_codes [
+          { code: 200, model: Entities::SpecialCase, message: "My special cases list" }
+        ]
+      end
+      get do
+        special_cases = SpecialCase.all.where(user: current_user, deleted: false)
```

Comment:
`SpecialCase.where(user: current_user, deleted: false)` should be enough, please have a try using the console.